settings-plugin/src/main/kotlin/com/freeletics/gradle/plugin/SettingsExtension.kt

```diff
@@ -4,7 +4,8 @@ import org.gradle.api.initialization.Settings
 
 abstract class SettingsExtension(private val settings: Settings) {
 
-    fun snapshots() {
+    @JvmOverloads
```

Comment:
nit: comment about parameter with a link

```suggestion
    /**
     * @param androidXBuildId   buildId for androidx snapshot artifacts. Can be taken from here:
     *                          https://androidx.dev/snapshots/builds
     */
    @JvmOverloads
```